client/src/js/base/SelectBox.js

```diff
@@ -1,11 +1,13 @@
 import styled from "styled-components";
 import { Box } from "./Box";
+import React from "react";
 
-export const SelectBox = styled(Box)`
+export const StyledSelectBox = styled(Box)`
     border: 1px ${props => (props.active ? props.theme.color.blue : props.theme.color.greyLight)} solid;
     border-radius: ${props => props.theme.borderRadius.sm};
     box-shadow: ${props => (props.active ? props.theme.boxShadow.inset : "none")};
-
+    background: white;
+    text-align: left;
```

Comment:
Why?

Reply:
Changes in formatting between the default styling of button and div. Without those lines the buttons look like this:
![Screenshot from 2021-10-01 09-16-02](https://user-images.githubusercontent.com/59776400/135656512-31d4066c-a9d4-4f33-af93-d5525bf2aaae.png)

I like how the original looks but I am happy to leave the button styling if preferred.